src/js/background.js

```diff
@@ -348,15 +414,25 @@ function Notifier() {
             priority: 1,
             progress: progress
         };
-        chrome.notifications.create("id_" + id, options, function(notificationId) {
-            if (onClicked) {
-                that.callbacks[notificationId] = onClicked;
-            }
-        });
+
+        if (progress == 100) {
+            options.buttons = [{ title: "Copy URL To Clipboard", iconUrl: "/icons/copy_18dp_2x.png" }];
+        }
+        if (progress > 0) {
+            chrome.notifications.update("prog_" + id, options, function(){
```

Comment:
Not exactly sure what you mean by that. Should we convert all the prefixes for notifications to caps?